### PR TITLE
Alpha to beta

### PR DESCRIPTION
**Pull request by aermakov-zalando**
* **Dev to alpha**
   <sup>Merge pull request #1644 from zalando-incubator/dev-to-alpha</sup>
* **Disable TaintNodesByCondition/ScheduleDaemonSetPods**
   <sup>Merge pull request #1639 from zalando-incubator/disable-stuff-back</sup>
* **Fix CoreDNS PDB**
   <sup>Merge pull request #1629 from zalando-incubator/fix-coredns-pdb</sup>
* **Dev to alpha**
   <sup>Merge pull request #1637 from zalando-incubator/dev-to-alpha</sup>
* **Worker nodes: don't enable useless feature gate**
   <sup>Merge pull request #1636 from zalando-incubator/revert-worker-change</sup>
* **Dev to alpha**
   <sup>Merge pull request #1633 from zalando-incubator/dev-to-alpha</sup>
* **Revert "enable east-west setup"**
   <sup>Merge pull request #1634 from zalando-incubator/revert-1565-enable/east-west</sup>
* **Enable ScheduleDaemonSetPods feature gate**
   <sup>Merge pull request #1628 from zalando-incubator/daemonset-scheduling</sup>
* **enable east-west setup**
   <sup>Merge pull request #1565 from zalando-incubator/enable/east-west</sup>
* **dev-to-alpha**
   <sup>Merge pull request #1626 from zalando-incubator/dev-to-alpha</sup>
* **Fix RBAC configuration for nvidia daemonsets**
   <sup>Merge pull request #1623 from zalando-incubator/fix-rbac-for-nvidia</sup>
* **Implement HTTP timeout for logging-agent**
   <sup>Merge pull request #1625 from zalando-incubator/update-logging-agent</sup>
* **Upgrade zmon-worker**
   <sup>Merge pull request #1622 from zalando-incubator/upgrade-zmon-worker</sup>
* **Separate LaunchTemplates by using node pool name**
   <sup>Merge pull request #1624 from zalando-incubator/launchtemplate</sup>
* **LaunchTemplates for worker node pools**
   <sup>Merge pull request #1619 from zalando-incubator/worker-nodepool-launchtemplate</sup>
* **dev-to-alpha**
   <sup>Merge pull request #1617 from zalando-incubator/dev-to-alpha</sup>
* **+ HighLoad++ talk**
   <sup>Merge pull request #1620 from zalando-incubator/highload-2018-video</sup>
* **Allow a port to be opened for CDP proxy**
   <sup>Merge pull request #1609 from zalando-incubator/open-cdp-proxy-port</sup>

### CODE REVIEW

**Comment by aermakov-zalando** [comment]
Update #1638 with new commits.

**Comment by aermakov-zalando** [comment]
👍

**Comment by jrake-revelant** [comment]
👍

**Comment by arjunrn** [comment]
:+1:

File: cluster/manifests/nvidia/nvidia-driver-installer.yaml [hunk 1]
```diff
@@ -22,6 +22,9 @@ spec:
         application: nvidia-driver-installer
         version: c117d39
     spec:
+{{ if index .ConfigItems "enable_rbac"}}
+      serviceAccountName: nvidia
```
Comment (dionb) [inline cluster/manifests/nvidia/nvidia-driver-installer.yaml:26]: Should this have an else?

File: cluster/manifests/nvidia/nvidia-gpu-device-plugin.yaml [hunk 2]
```diff
@@ -22,6 +22,9 @@ spec:
         application: nvidia-gpu-device-plugin
         version: cb2bdea5eb507a9577ce11ff058a2dfa84f54263
     spec:
+{{ if index .ConfigItems "enable_rbac"}}
+      serviceAccountName: nvidia
```
Comment (dionb) [inline cluster/manifests/nvidia/nvidia-gpu-device-plugin.yaml:26]: Same here. Should this have an else?

File: cluster/manifests/nvidia/nvidia-driver-installer.yaml (hunk 1 above)
Comment (aermakov-zalando) [reply cluster/manifests/nvidia/nvidia-driver-installer.yaml:26]: Yes, but this is wrong in multiple places so we can fix it in a separate PR.

File: cluster/manifests/nvidia/nvidia-gpu-device-plugin.yaml (hunk 2 above)
Comment (aermakov-zalando) [reply cluster/manifests/nvidia/nvidia-gpu-device-plugin.yaml:26]: Same here.

**Pull request merged**